TEAMS.yaml: add convenience tool advice for finding triage_column_id

otan opened the pull request:
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/64361)
<!-- Reviewable:end -->

tbg reviewed (approved):
Cool stuff! A few questions that the commit message could answer:

- why is this a separate tool in the `cockroachlabs` repo?
- can we in the future, theoretically, invoke it as a library, and refer to the triage column by string in TEAMS.yaml? Not suggesting we do this now or, ever.

otan committed:
TEAMS.yaml: add convenience tool advice for finding triage_column_id

I've added a [tool](github.com/cockroachlabs/github-find-triage-column-id)
which handles finding the triage_column_id for TEAMS.yaml. This is a
separate repo as it pollutes the vendoring of the cockroachdb repo and
prefer it not being part of bors.

We could, in theory, make the TEAMS.yaml fetch this every time and make
the user specify the org, repo, project and column name instead.
However, this can be very unstable when columns are renamed (it's happened
to blathers). Maybe if we decide it's worthwhile in future.

Release note: None

otan commented:
bors r=tbg
